pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "Etna"
-version = "6.3.0"
+version = "7.1.0"

```

Comment:
Hi @jamesbiggs, this number is matching the current Etna release. Would change in the version apply for every etna release regardless of toml/lock file change or whenever there is a change specifically to toml/lock file?

Reply:
Hey Allan, I've been updating this every time I update packages - but making it the current project release number. 6.3.0 was the last time I updated packages, and as this isn't massively important I hadn't done it since 6.3.0 :) But this should show the current version of our project 🙌

Reply:
Hey Allan, I've been updating this every time I update packages - but making it the current project release number. 6.3.0 was the last time I updated packages, and as this isn't massively important I hadn't done it since 6.3.0 :) But this should show the current version of our project 🙌

Reply:
Hi @jamesbiggs , thanks for letting me know and bring it to the current level with the release, in that case I can skip this from future reviews.